r/R/type.R

```diff
@@ -187,8 +187,11 @@ NestedType <- R6Class("NestedType", inherit = DataType)
 #' @param timezone For `timestamp()`, an optional time zone string.
 #' @param byte_width byte width for `FixedSizeBinary` type.
 #' @param list_size list size for `FixedSizeList` type.
-#' @param precision For `decimal()`, precision
-#' @param scale For `decimal()`, scale
+#' @param precision For `decimal()`, the number of significant digits
+#'    the arrow `decimal` type can represent. Currently `decimal()` is mapped
+#'    to `DecimalType128`, having a maximum precision of 38 significant digits.
+#' @param scale For `decimal()`, the number of digits after the decimal
+#'    point. It can be negative.
```

Comment:
> It can be negative. 

Would it make sense to expand on why one might want to do this? The argument text itself is probably not the right place, but adding a paragraph above in the description might be nice.

Reply:
I was debating where to add more info. I agree the arg description isn't the place. Wouldn't it get a bit lost if we put it in the more general `data-type` description?